src/Login.ts

```diff
@@ -104,6 +104,14 @@ export class Login {
     return url;
   }
 
+  public static stopAutoAuthorize(authResult: AuthResult): void {
```

Comment:
Why not just accessToken?

Reply:
Yes I could but then we can't use WeakMap but maybe that is ok?

Reply:
I will take a look